apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsAnyOfControl.tsx

```diff
@@ -25,27 +25,27 @@ export function JsonFormsOneOfControl({
   uischema,
   uischemas,
   label,
-  description,
 }: CombinatorRendererProps) {
-  const oneOfRenderInfos = createCombinatorRenderInfos(
-    schema.oneOf || [],
+  const anyOfRenderInfos = createCombinatorRenderInfos(
+    schema.anyOf || [],
     rootSchema,
-    'oneOf',
+    'anyOf',
```

Comment:
to clarify this actually is a different logic - `oneOf` allows exactly one of a list of options whereas `anyOf` allows >=1 of the options within a list of allowed options

Reply:
But here it is fine since we ensure that we only allow 1 value from the UI perspective?

Reply:
Yup, it is more of because we will always have a constant that differentiates between the schemas. For example, we use the `variant` key to differentiate between the different variants in the Hero component, so it is effectively operating with a `oneOf`.